2_process/src/pair_nhd_reaches.R

```diff
@@ -1,21 +1,22 @@
+#' @description Function to pair PRMS segments with associated NHDPlusV2 flowlines
```

Comment:
Just noting that some other functions in this repo have these descriptions within the function's { }. I'm indifferent to the location, though.

Reply:
Yeah, I'm slowly migrating some of this formatting so that the descriptions are outside the function's {}. Thanks for pointing this out though!